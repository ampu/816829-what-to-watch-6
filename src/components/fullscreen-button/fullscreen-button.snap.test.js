import React from 'react';

import {render} from '@testing-library/react';

import FullscreenButton from './fullscreen-button';

describe(`FullscreenButton component`, () => {
  it(`Should render according to the snapshot`, () => {
    const {container} = render(<FullscreenButton/>);
    expect(container).toMatchSnapshot();
  });
});
