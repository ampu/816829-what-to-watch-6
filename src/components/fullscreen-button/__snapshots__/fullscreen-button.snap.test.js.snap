// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullscreenButton component Should render according to the snapshot 1`] = `
<div>
  <button
    type="button"
  >
    <svg
      height="27"
      viewBox="0 0 27 27"
      width="27"
    >
      <use
        xlink:href="#full-screen"
      />
    </svg>
    <span>
      Full screen
    </span>
  </button>
</div>
`;
